test/unit/test_deploy.py

```diff
@@ -1,15 +1,17 @@
 import fnmatch
 import logging
 import os
-
 import pandas
 from nose.plugins.attrib import attr
 
-from ..test_base import AssetManagementUnitTest
+from test.test_base import AssetManagementUnitTest
+
+csv_file = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'reference_designators.csv')
 
 logging.basicConfig()
 log = logging.getLogger()
 log.setLevel(logging.INFO)
+rd_set = set(pandas.read_csv(csv_file)['reference_designator'].tolist())
```

Comment:
this can be simplified to:

`set(pandas.read_csv(csv_file).reference_designator)`

I recommend you move this to the setUpClass in test_base.py, then if we ever need it for any of the other test classes it is already available.